kernel-modules/BLOCKLIST

```diff
@@ -66,3 +66,7 @@
 6.2.* 2.2.0 *
 6.2.* 2.1.0 *
 6.2.* 2.0.1 *
+
```

Comment:
I would not leave an empty line here, should be harmless, but may also not be harmless (pick performance review comment, I know).